pootle/static/js/common.js

```diff
@@ -203,6 +205,16 @@ PTL.common = {
       });
     });
 
+
+    /* Bind hotkeys */
+    const hotkeys = mousetrap(document.body);
+
+    hotkeys.bind('f1', (e) => {
```

Comment:
<kbd>⌘</kbd>+<kbd>?</kbd> is reserved for the OS' help menu and I wouldn't resource to using <kbd>Ctrl</kbd> at all as it's not in line with the platform.

To be honest, I don't think there's anything wrong with mapping it to <kbd>?</kbd> additionaly, even if it cannot be triggered from a focused editor.